app/controllers/actions/IdentifierAction.scala

```diff
@@ -42,7 +42,7 @@ class AuthenticatedIdentifierAction @Inject()(override val authConnector: AuthCo
     val retrievals = Retrievals.internalId and
                      Retrievals.affinityGroup
 
-    implicit val hc : HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers)
+    implicit val hc : HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
```

Comment:
I believe this one should be `HeaderCarrierConverter.fromRequestAndSession` as per https://github.com/hmrc/http-verbs/blob/master/CHANGELOG.md#version-1300

Reply:
Done